### DISCUSSION TITLE
Deta cli is bad

### DISCUSSION THREAD

**Discussion by whzinn** [deta/cloud-docs#742] [Ideas]
Eu não entendi por que um serviço que devia ser de fácil deploy tem um cli próprio que precisa ser estudado, onde o deploy deveria ser por um arquivo zip pra facilitar a vida ou usar o Github pra deploy como as outras plataformas, menos é mais certo

**Comment by lemonyte** [comment]
I see nothing difficult in typing a single command to deploy a project. Zipping your project contents every time you wish to re-deploy would certainly be much more time consuming. You may use GitHub to deploy projects, using an action such as [Deploy to Deta](https://github.com/marketplace/actions/deploy-to-deta) or [Deta Space Deployment Github Action](https://github.com/marketplace/actions/deta-space-deployment-github-action). Perhaps deploying from a ZIP file would be a useful feature for some people, but your post would have been better presented as a suggestion rather than an opinion of "Deta cli is bad". Although all feedback is valuable, please remember to be respectful of the community and the people who provide the services you are using free of charge.

###### Note: written in response to English translation of original post in Portuguese

**Reply by whzinn** [reply]
I not uses deta little friend, the title is catchy, why does a free service have a cli and no github, and how is the deta sustainable? it is no longer accusing the only way for deta to work seriously a kubernetes cluster network with zombie computers #
